src/components/filters/FilterMenu.jsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import Sheet from 'react-modal-sheet';
+import Checkbox from "./FilterCheckbox";
+import { XIcon } from "../icons/Icons";
+
+export function FilterMenuDesktop({ filters, setFilters }) {
+    function clearFilters() {
+        let clist = document.getElementsByTagName("input");
+        for (let i = 0; i < clist.length; ++i) { clist[i].checked = false; }
+        setFilters((prev) => ({...prev, hobo: false }))
```

Comment:
This may be hard to scale with this pattern, Defer to @alexsaltstein on how we want handle this going forward

Reply:
prob can just do a for loop on each key of the filters object